tests/baseline/test_baseline.py

```diff
@@ -77,6 +81,24 @@ def koji_test_backend(fake_koji, koji_dir):
     Source.reset()
 
 
+@pytest.fixture(autouse=True)
+def fake_kerberos_auth(mocker):
+    mocker.patch("gssapi.Name")
+    mocker.patch("gssapi.Credentials.acquire")
+    mocker.patch("requests_gssapi.HTTPSPNEGOAuth", return_value=None)
+
+
+@pytest.fixture(autouse=True)
+def errata_requests_mock(requests_mock):
+    with open(ERRATUM_DATA_FILE, "r") as f:
+        advisory_return_val = json.load(f)
+
+    requests_mock.get(
+        "/api/v1/erratum/RHBA-2020:2807", json=advisory_return_val
```

Comment:
Why doesn't the ID in the URL match the ID in the data?

I think the chosen testdata makes things unnecessarily confusing, e.g. we are seeing e2e product name appear against obviously non-e2e images in the test output.  That makes it harder to tell: is the test output weird because the code is buggy, or is it weird because the test input is unrealistic.  Currently it's the latter but this seems unnecessary, the real data could have been used.

Reply:
Added the correct data.